instagram-page-react/src/app.constants/index.js

```diff
@@ -0,0 +1,9 @@
+const EMPTY_ARRAY = Object.freeze([]);
+const EMPTY_STRING = '';
```

Comment:
String is already a primitive type, so we don't need this constant
It is the same as defining a constant like `const ZERO = 0;`